spk/mc/src/service-setup.sh

```diff
@@ -2,4 +2,18 @@ service_postinst ()
 {
     # Use the mc-utf8 command for mc
     ln -s ${SYNOPKG_PKGDEST}/bin/mc-utf8 /usr/local/bin/mc
-}
\ No newline at end of file
+    ln -s ${SYNOPKG_PKGDEST}/bin/mc-utf8 /usr/local/bin/mcedit
+    ln -s ${SYNOPKG_PKGDEST}/bin/mc-utf8 /usr/local/bin/mcdiff
+    ln -s ${SYNOPKG_PKGDEST}/bin/mc-utf8 /usr/local/bin/mcview
+}
+
+service_postuninst ()
+{
+     # Remove symlinks
+     rm -f /usr/local/bin/mc
+     rm -f /usr/local/bin/mcedit
+     rm -f /usr/local/bin/mcdiff
+     rm -f /usr/local/bin/mcview
+
+     exit 0
```

Comment:
Please remove `exit 0` which will prevents other steps from generic installer to be executed.